front/src/modules/companies/components/AddPersonToCompany.tsx

```diff
@@ -121,20 +139,32 @@ export const AddPersonToCompany = ({
 
   return (
     <RecoilScope>
-      <StyledContainer>
-        <div ref={refs.setReference}>
-          <LightIconButton
-            Icon={IconPlus}
-            onClick={handleOpenPicker}
-            size="small"
-            accent="tertiary"
-          />
-        </div>
+      <StyledContainer ref={refs.setReference}>
+        <LightIconButton
+          Icon={IconPlus}
+          onClick={handleOpenPicker}
+          size="small"
+          accent="tertiary"
+        />
 
         {isDropdownOpen && (
           <div ref={refs.setFloating} style={floatingStyles}>
             {isCreationDropdownOpen ? (
               <StyledInputContainer>
+                <HotkeyEffect
```

Comment:
here same, I think that if you use onCancel from PeoplePicker and onEscape from DoubleTextFieldInput, you should not have to register this hotkeyEffect here :)

Reply:
Update